src/homepageExperience/components/steps/go/InitalizeClient.tsx

```diff
@@ -40,9 +40,11 @@ func main() {
       </p>
       <CodeSnippet text={codeSnippet} onCopy={logCopyCodeSnippet} />
       <p style={{marginTop: '42px'}}>
-        Here, we initialize the token, organization info, and server url that is
-        needed to set up the initial connection to InfluxDB. The client
-        connection is then established with InfluxDBClient initialization.
+        Here, we initialize the token, organization info, and server url that
+        are needed to set up the initial connection to InfluxDB. The client
+        connection is then established with the{' '}
+        <code className="homepage-wizard--code-highlight">influxdb2</code>{' '}
+        initialization.
```

Comment:
`Here, we initialize the token, organization information, and server URL that are needed to set up the initial connection to InfluxDB. The client connection is then established with InfluxDB initialization.`

Amy sent me that for this change. It's just slightly different.

Reply:
I think I had slightly more context on it than she did - we want the text mentioning the initialization to match the library being used. For Ptyhon it's `InfluxDBClient`, for go it's `influxdb2`, for nodejs it's `InfluxDB`. I added in the `the` because it makes more grammatical sense, since it's referring to a specific library